pkg/dartdev/lib/src/templates/console.dart

```diff
@@ -44,7 +44,8 @@ version: 1.0.0
 environment:
   ${common.sdkConstraint}
 
-# dependencies:
+dependencies:
```

Comment:
👍  - this is a good change; this will probably also work better w/ the `dart pub add` command.

Two nits; the comment between `dependencies` and the example entry looks odd - it may look better before the dependencies section. And, we have a few more of these commented out dependencies sections in the other templates.

Reply:
Perhaps https://dart-review.git.corp.google.com/c/sdk/+/273543 instead?

Reply:
That Gerrit CL has landed, and I think it replaces this PR, so I am tentatively closing this PR.  If it isn't reopened, then I will remove the branch for this PR as well.